### PR TITLE
Add Users table migration.

### PR DESCRIPTION
**Pull request by Gallup93**
Add Users table with attributes:
-name
-address
-city
-state
-zip
-email
-password
-role
-timestamps

### CODE REVIEW

File: db/migrate/20200527004935_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    create_table :users do |t|
+      t.string :name
+      t.string :address
+      t.string :city
+      t.string :state
+      t.string :zip
+      t.string :email
+      t.string :password_digest
+      t.integer :role
+
```
Comment (stellakunzang) [inline db/migrate/20200527004935_create_users.rb:12]: THIS IS SO GOOOD

**Comment by stellakunzang** [comment]
https://gph.is/1sDiaKy

File: db/schema.rb [hunk 2]
```diff
@@ -67,6 +67,19 @@
     t.index ["item_id"], name: "index_reviews_on_item_id"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "city"
```
Comment (sagemlee) [inline db/schema.rb:73]: Looks great, glad we picked the one migration that hadnt been done.

**Review by sagemlee — APPROVED** [review]

**Review by stellakunzang — APPROVED** [review]
Approved!

**Review by muydanny — APPROVED** [review]

**Pull request merged**